cli/cmd/install.go

```diff
@@ -122,6 +123,14 @@ A full list of configurable values can be found at https://www.github.com/linker
 			if err != nil {
 				return err
 			}
+
+			// Create values override
+			valuesOverrides, err := options.MergeValues(nil)
+			if err != nil {
+				return err
+			}
+
+			var k8sAPI *k8s.KubernetesAPI
```

Comment:
any reason we need to pre-declare this outside the block?  can we just := on line 147?